use/extending.md

```diff
@@ -52,6 +52,7 @@ If you try to source install the `curl` package without libcurl, the installatio
 
 In general, system requirements can be found on error messages on installation failures, the package's reference page,
 CRAN, or [METACRAN](https://www.r-pkg.org/).
+[The dashboard of system libraries linked by R packages on R-universe](https://r-universe.dev/sysdeps/) is useful as well.
```

Comment:
You could mention r2u as well. It has them resolved as it relies on `ldd` output along with the knowledge the distro has about its packages.

Reply:
I think It is great that there is a reference to it.
But I still do not understand how r2u works.
Since you can edit this branch too, I was wondering if you could add a commit to it, if you don't mind.
(I'm going to bed now and won't be editing for the rest of the day)

Of course you or I can edit this page later than this PR.

Reply:
r2u is "simply" two things: a complete repository for both Ubuntu 20.04 and 22.04 covering all of CRAN, and just over 200 BioC packages.   So all `apt install ...` works as you expect.  It combines this with `bspm` which then maps R's own `install.packages()` into the corresponding `apt` calls so that `install.packages(c("tidyverse", "sf", "brms"))` does what you expect.  And it does what no other system for Ubuntu does: all binaries, with full (distro-level) resolution.  Happy to talk more (and I should probably bring the two `r2u` container setups into Rocker but I didn't want to be too pushy about it.

Reply:
(And yes I really need to document the various r-base related containers better too.)

Reply:
It seems difficult for me to explain the difference between r2u and a traditional installation from PPA...

Reply:
The key difference are where packages comes from:
- The Rutter PPA is on Launchpad and provides ~ 5k package.  
- Whereas r2u runs from my server and has ~ 19k packages.

Reply:
OK, thanks to opening rocker-org/rocker#496.